sympy/matrices/tests/test_matrices.py

```diff
@@ -244,8 +244,6 @@ def test_power():
     assert A**10.2 == Matrix([[0, 0, 1], [0, 0, 1], [0, 0, 1]])
     A = Matrix([[0, 1, 0], [0, 0, 1], [0, 0, 1]])  # Nilpotent jordan block size 2
     assert A**10.0 == Matrix([[0, 0, 1], [0, 0, 1], [0, 0, 1]])
-    n = Symbol('n', integer=True)
-    raises(ValueError, lambda: A**n)
```

Comment:
This should be replaced with a test that it doesn't raise an exception. Maybe `assert isinstance(A**n, MatPow)`.